__tests__/extension-requests/extension-requests.test.js

```diff
@@ -358,6 +358,25 @@ describe('Tests the Extension Requests Screen', () => {
           },
           body: JSON.stringify(extensionRequestsListUserSearch),
         });
+      } else if (
+        url ===
+        'https://api.realdevsquad.com/extension-requests?order=asc&size=1&q=status%3APENDING'
```

Comment:
why this api url is hardcoded

Reply:
This is because for my test, I'm fetching that specific request's mock data from extensionRequestsListPending which is mocked to be successfully approved/denied. Some of the other requests, in extensionRequestsListPending are mocked to fail.

Reply:
> This is because for my test, I'm fetching that specific request's mock data from extensionRequestsListPending which is mocked to be successfully approved/denied. Some of the other requests, in extensionRequestsListPending are mocked to fail.

reason seems not correct to me, please confirm from @prakashchoudhary07